colour/utilities/array.py

```diff
@@ -677,3 +732,82 @@ def linear_conversion(a, old_range, new_range):
     out_min, out_max = tsplit(new_range)
 
     return ((a - in_min) / (in_max - in_min)) * (out_max - out_min) + out_min
+
+
+def fill_nan(a, method='Interpolation', default=0):
+    """
+    Fills given array NaNs accordingly to given method.
```

Comment:
Should this be `according to`? I think I saw this some other places, too.

Reply:
I will do that post-merge on the *v0.3.11* branch, adding a note in Asana about it.